History.md

```diff
@@ -1,3 +1,26 @@
+unreleased
+==========
+
+  * deps: body-parser@1.18.3
+    - Fix stack trace for strict json parse error
+    - deps: depd@~1.1.2
+    - deps: http-errors@~1.6.3
+    - deps: iconv-lite@0.4.23
+    - deps: qs@6.5.2
+    - deps: raw-body@2.3.3
+    - deps: type-is@~1.6.16
+  * deps: http-errors@~1.6.3
+    - deps: depd@~1.1.2
+    - deps: setprototypeof@1.1.0
+    - deps: statuses@'>= 1.3.1 < 2'
+  * deps: iconv-lite@0.4.23
+    - Fix loading encoding with year appended
+    - Fix deprecation warnings on Node.js 10+
+    - deps: safer-buffer@'>= 2.1.2 < 3'
+  * deps: raw-body@2.3.3
```

Comment:
There is no raw-body version bump in our package.json to correspond with this. Was the package.json change missed?